source/settings/core.rst

```diff
@@ -809,6 +809,25 @@ Example Setting:
    deliver_log_format = stime=%{session_time} msgid=%m: %$
 
 
+ .. _setting-dict_db_config:
+
+``dict_db_config``
+
+ - Default: <empty>
+ - Values: :ref:`string`
+
+Points to a Berkeley DB config file. Equivalent to adding
+DB_CONFIG=/path to import_environment.
+
+See https://web.stanford.edu/class/cs276a/projects/docs/berkeleydb/ref/env/db_config.html
```

Comment:
That seems a rather random URL. Some Stanford university class? Maybe https://docs.oracle.com/database/bdb181/html/bdb-sql/sql_db_config.html is more official?